server/container_create_linux.go

```diff
@@ -1008,7 +1035,7 @@ func addOCIBindMounts(mountLabel string, containerConfig *pb.ContainerConfig, sp
 		if mount.Readonly {
 			options = []string{"ro"}
 		}
-		options = append(options, "rbind", "nodev")
+		options = append(options, "rbind")
```

Comment:
I think you want nodev to be everywhere except if the mountpoint is from /dev.  But I think this is too big of a hammer.
@umohnani8 Did you add this?

Reply:
it's indeed to big, I've see nodev only in tmpfs mount in docker for instance. I don't know why we would add it everywhere

Reply:
Yeah I did, it was to fix the issue reported in this BZ https://bugzilla.redhat.com/show_bug.cgi?id=1627633.

Reply:
So this should be reverted to keep "nodev" there right?

Reply:
most likely, yeah, I'll will do that in a bit

Reply:
see https://github.com/kubernetes-sigs/cri-o/pull/1884/files#diff-aab4c88107335f18845d9e317503a02dR1040

we now add `nodev` if and only if the mount destination is not under `/dev/` (which would defeat the whole dev mount otherwise)